### ISSUE TITLE
Agree best place and name for "parent" field and update each content format - DUPLICATE

### ISSUE THREAD

**Issue by willguv** [localgovdrupal/localgov#511]
The location of the "parent" field varies from one content format to another

- On service pages, it's at the top
- On directory pages, it's second
- On guides and step by step, it's last

The name of the field also changes

- Parent
- Services: Parent

Let's decide where this field should live and what it should be called, and make changes to all formats

**Comment by keelanfh** [comment]
Very similar to #487. We've done a bunch of work on this at Essex, but it would need to be recreated in plain LocalGov Drupal because we've got some additional fields too.

**Comment by willguv** [comment]
@keelanfh would you be interested in running a special session for the Backlog group to run through all the admin interface changes you’ve made or are planning?

There’s been some longstanding issues I’ve wanted to fix, and we’ve never reviewed admin for usability. Sounds like you’re doing just that :)

**Comment by keelanfh** [comment]
Yeah, could do something like that. We've only made a few changes, though I think they do have a significant impact.

We've actually pivoted away from doing user research into the CMS side of things right now, as we have such a volume of things that have come up from earlier user research/ UX review, and we don't have capacity to act on them. But happy to bring things to the group if helpful - others might be in a better position to take things to the next stage.
